bot/cogs/snakes.py

```diff
@@ -41,8 +65,20 @@ def __init__(self, bot: AutoShardedBot):
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
 
-    # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
+        data = await self.get_snek(name)
+        await ctx.send(data)
+
+    # The snake moult command
 
+    @command()
+    async def moult(self, ctx: Context):
+        await ctx.send("sssss... moulting in progress...")
+        green = discord.utils.get(ctx.guild.roles, name="Green Skin")
+        black = discord.utils.get(ctx.guild.roles, name="Black Skin")
+        yellow = discord.utils.get(ctx.guild.roles, name="Yellow Skin")
+        await ctx.guild.me.remove_roles()
```

Comment:
This will remove _all_ the roles, won't it? Maybe specify the roles that should be removed.

Reply:
Ill get right on fixing. :)